components/molecules/ErrorAlert.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div>
+        <div v-if="hasConflict">
+            <div :class="baseClass" class="error-alert" role="alert">
+                <p>{{ conflictMessage }}</p>
+                <button @click="onClose">
+                    <img alt="close" src="/img/close.svg" width="16" />
+                </button>
+            </div>
+        </div>
+        <div v-else-if="hasError">
+            <div :class="baseClass" class="error-alert" role="alert">
+                <p>
+                    {{ errorMessage }}
+                    <b-link v-if="errorLink != null" :to="errorLink.link" class="error-alert-link">
+                        {{ errorLink.text }}
+                    </b-link>
+                </p>
+                <button @click="onClose">
+                    <img alt="close" src="/img/close.svg" width="16" />
+                </button>
+            </div>
+        </div>
+    </div>
+</template>
+<script lang="ts" setup>
+import { computed, ComputedRef, PropType } from '@nuxtjs/composition-api'
+import { useStores } from '~/composables/useStores'
+import { ErrorLink } from '~/local/models/ErrorLink'
+
+const props = defineProps({
+    baseClass: {
+        type: String,
```

Comment:
Do we need to specify type if we are setting a default value?

Reply:
> Do we need to specify type if we are setting a default value?

We see it as good practice.